Tests/store_packs.py

```diff
@@ -0,0 +1,512 @@
+import json
+import os
+import sys
+import argparse
+import warnings
+import shutil
+import uuid
+import google.auth
+from google.cloud import storage
+from distutils.util import strtobool
+from distutils.version import LooseVersion
+from datetime import datetime
+from zipfile import ZipFile, ZIP_DEFLATED
+from Tests.test_utils import run_command, print_error, print_warning, print_color, LOG_COLORS, \
+    collect_content_items_data, input_to_list
+
+# global constants
+STORAGE_BASE_PATH = "content/packs"
+IGNORED_PATHS = ['Packs/__init__.py']
+# the format is defined in issue #19786, may change in the future
+DIR_NAME_TO_CONTENT_TYPE = {
+    "Classifiers": "classifier",
+    "Dashboards": "dashboard",
+    "IncidentFields": "incidentfield",
+    "IncidentTypes": "incidenttype",
+    # "IndicatorFields": "indicatorfields",
```

Comment:
@IkaDemisto are we removing this or not?

Reply:
we will include it as `reputation` field.